client/src/components/new-request/search.vue

```diff
@@ -30,7 +30,7 @@
                   <v-tooltip :disabled="!data.item.blurbs" right transition="fade-transition">
                     <template v-slot:activator="scope">
                       <span v-on="scope.on" class="list-item">
-                        {{ displayRequestActionListItem(data.item) }}
+                        {{ data.item.text }}
```

Comment:
Why did you revert this change?

Reply:
it was a typo in the UX design to begin with :P

Reply:
The design had it dynamically changing from dropdown list to selection, but it was just a mistake